profile/README.md

```diff
@@ -0,0 +1,37 @@
+### Welcome to Flotiq Github page :raised_hand_with_fingers_splayed:
+
+<img src="https://flotiq.com/static/hero_desktop-4894423ad4df154677959c0b251aabad.svg" alt="Flotiq baner" title="Flotiq"/>
+
+<p align="center">
+    <img src="https://skillicons.dev/icons?i=kubernetes,docker,react,nodejs,tailwind" />
+</p>
+
+## API first headless CMS 
+
+### Enjoy fast and painless content delivery to any channel with our headless CMS platform.
+
+**[Try live demo](https://meet.simplyscale.io/15min-flotiq-tour)**
```

Comment:
This is not a link to "live demo". This is a link to "book a 15 minute tour of the system".